ui/app/AppLayouts/Profile/ProfileLayout.qml

```diff
@@ -193,7 +194,7 @@ StatusSectionLayout {
                 implicitHeight: parent.height
 
                 languageStore: root.store.languageStore
-                currencyStore: root.store.walletStore.currencyStore
+                currencyStore: SharedStores.RootStore.currencyStore
```

Comment:
not necessarily against it, but how come?

Reply:
To fix this:
`Warning: Cyclic dependency detected between "qrc:/imports/shared/stores/RootStore.qml" and "qrc:/app/AppLayouts/Profile/stores/WalletStore.qml" (:0, )`

I've removed this line the was causing the warning from qrc:/app/AppLayouts/Profile/stores/WalletStore.qml:
    `property var currencyStore: SharedStore.RootStore.currencyStore`
As a result I needed to update other components that were previously using WalletStore.currencyStore to use RootStore.currencyStore. The data was the same, just passed from one store to another.